File: app/src/main/java/com/theappsolutions/boilerplate/util/other_utils/IntentUtils.java
package com.theappsolutions.boilerplate.util.other_utils;

import android.content.Context;
import android.content.Intent;
import android.net.Uri;

/**
 * @author Dmytro Yakovlev d.yakovlev@theappsolutions.com
 * @copyright (c) 2017 TheAppSolutions. (https://theappsolutions.com)
 */
public class IntentUtils {

    public static void openLinkInBrowser(Context context, String url) {
        Intent intent = new Intent(Intent.ACTION_VIEW);
        intent.setData(Uri.parse(url));

        if (intent.resolveActivity(context.getPackageManager()) != null) {
            context.startActivity(intent);
        }
    }

}

File: app/src/main/java/com/theappsolutions/boilerplate/util/other/IntentUtils.kt
package com.theappsolutions.boilerplate.util.other

import android.content.Context
import android.content.Intent
import android.net.Uri

/**
 * @author Dmytro Yakovlev d.yakovlev@theappsolutions.com
 * @copyright (c) 2017 TheAppSolutions. (https://theappsolutions.com)
 */
    fun openLinkInBrowser(context: Context, url: String) {
        val intent = Intent(Intent.ACTION_VIEW)
        intent.data = Uri.parse(url)

        if (intent.resolveActivity(context.packageManager) != null) {
            context.startActivity(intent)
        }
}
